ateliermecanique-ws/build.gradle

```diff
@@ -69,6 +69,11 @@ dependencies {
     implementation('com.h2database:h2')
 
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'	//for serializing and deserializing java.time.LocalDateTime
+    implementation group: 'com.sun.mail', name: 'javax.mail', version: '1.6.2' //for mailing service
+    implementation 'org.apache.httpcomponents:httpcore:4.4.15' // for mailing service
+    implementation('org.apache.httpcomponents:httpclient:4.5.14') //for mailing service
+    implementation 'org.springframework.boot:spring-boot-starter-mail' //for mailing service
```

Comment:
Thanks for organizing your newly added dependencies Mohit. I could imagine things like this are appreciated greatly in the industry